content/blog/announcing_vmware_as_gold_sponsor.md

```diff
@@ -0,0 +1,19 @@
+Title: Announcing VMware as Gold Sponsor
+Date: 2018-09-29 20:00
```

Comment:
Can you update the date to be earlier. Probably 01:00 since future posts don't show up

Reply:
Fixed.